doc/teknologiat.md

```diff
@@ -10,19 +10,19 @@ Ajoympäristö: https://github.com/finnishtransportagency/geoviite/blob/main/aws
 
 | Backend                       | Teknologia              | Versio |
 |-------------------------------|-------------------------|--------|
-| Totetuskieli                  | Kotlin                  | 1.4    |
```

Comment:
Näitä ei oltu toviin päivitelty. Vähän höhelö dokumentaatio kun oikeat versionumerot on tietenkin npm/gradle conffissa, mutta tätä muotoa jossain kohtaa kaipailtiin